src/components/notifications/AppExplorer/AppCard/index.tsx

```diff
@@ -55,6 +56,8 @@ const AppCard: React.FC<AppCardProps> = ({ name, description, logo, bgColor, url
     [userPubkey, name, description, logo, bgColor, url]
   )
 
+  const imgRef = React.useRef(null)
```

Comment:
What are we using the ref for

Reply:
Oops, my bad, was trying something else with this, will remove this